imageroot/actions/create-module/90firewall

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import agent
+import os
+
+agent.add_public_service(os.environ['MODULE_ID'], [
```

Comment:
Is it safe/necessary to always open the Piler SMTP port to the world?

If a NS8 node has to provide this email archive service to an external mail server, maybe a restricted IP policy would be needed too.

I'd rather just document how to manually configure the firewall for that, and remove Firewalld API calls.

Reply:
yes this is how we could accept archive email to piler, other method could be to use an import tools but it will be by a cron and we need to know what account to export to the archive.

Piler listens only on the email archive@domain.com

Reply:
I agree with Steph: the port must be open to receive the messages.

We do not have a firewall policy based on IP, nor we decided to implement it.
Even this decision can be changed, I think the authorization part should be built inside the app,

In this case, we can use SMTP ACLs: https://mailpiler.com/smtp-acl-list/

Reply:
Ok for SMTP ACLs, however opening the random high port is an half-baked solution: an external ns7 mail server still needs a template-custom to work with it :unamused: 

I'd prefer to avoid the "external mail server" scenario for now, unless binding to port 25 is added here too.

Reply:
>an external ns7 mail server still needs a template-custom to work with it 

Sorry I do not understand, why to not simply use 

```
db smarthosts set @piler.domain.com recipient Host piler.domain.com Password '' Port 20010 TlsStatus enabled Username '' status enabled
signal-event nethserver-mail-smarthosts-save
```

Reply:
Sadly ACL cannot block or allow IPV6